pkg/aggregator/aggregator_test.go

```diff
@@ -257,3 +264,58 @@ func TestRecurentSeries(t *testing.T) {
 	s.AssertNotCalled(t, "SendSketch")
 
 }
+
+func TestTags(t *testing.T) {
+	version.AgentVersion = "7"
```

Comment:
nit: you can `defer` setting this back to its original value

Reply:
this is actually set at compile time so we can't really guess the real value, I can use it directly though to build the expected tag in the unit tests: `[]string{"version:"+version.AgentVersion}`  instead of `[]string{"version:7"}`